src/osd/scheduler/mClockScheduler.cc

```diff
@@ -57,6 +141,7 @@ mClockScheduler::mClockScheduler(CephContext *cct,
   set_config_defaults_from_profile();
   client_registry.update_from_config(
     cct->_conf, osd_bandwidth_capacity_per_shard);
+  _init_logger();
```

Comment:
This causes the unit test to fail. Please incorporate logger into the unit test to fix it. Will remove the qa tag for until it's addressed.

Reply:
ok